ngraph/test/frontend/paddlepaddle/op.cpp

```diff
@@ -96,46 +96,25 @@ inline void visualizer(std::shared_ptr<ngraph::Function> function, std::string p
     network.serialize(path + ".xml", path + ".bin");
 }
 
-std::string get_npy_dtype(std::string& filename)
-{
-    std::ifstream stream(filename, std::ifstream::binary);
-    if (!stream)
-    {
-        throw std::runtime_error("io error: failed to open a file.");
-    }
-
-    std::string header = npy::read_header(stream);
-
-    // parse header
-    npy::header_t npy_header = npy::parse_header(header);
-    return npy_header.dtype.str();
-}
-
-template <typename T>
-void load_from_npy(std::string& file_path, std::vector<T>& npy_data)
-{
-    std::ifstream npy_file(file_path);
-    std::vector<unsigned long> npy_shape;
-    bool fortran_order = false;
-    if (npy_file.good())
-        npy::LoadArrayFromNumpy(file_path, npy_shape, fortran_order, npy_data);
-
-    if (npy_data.empty())
-    {
-        throw std::runtime_error("failed to load npy for test case " + file_path);
-    }
-}
-
 namespace fuzzyOp
 {
     using PDPDFuzzyOpTest = FrontEndBasicTest;
     using PDPDFuzzyOpTestParam = std::tuple<std::string,  // FrontEnd name
                                             std::string,  // Base path to models
                                             std::string>; // modelname
-
-    void run_fuzzy(std::shared_ptr<ngraph::Function> function, std::string& modelfile)
+    template<typename T>
+    inline void add_input_output(cnpy::NpyArray& npy_array, test::TestCase<TestEngine>& test_case, bool is_input=true)
     {
-        auto modelfolder = get_modelfolder(modelfile);
+        T* npy_begin = npy_array.data<T>();
+        std::vector<T> data(npy_begin, npy_begin + npy_array.num_vals);
+        if(is_input)
```

Comment:
I see that 'clang' was not run on this file. Please rebase to latest 'pdpd_frontend' branch, it has proper 'clang' enabled for test sources.

Reply:
Will rebase the code